pkg/glusterfs/controllerserver.go

```diff
@@ -314,19 +297,20 @@ func (cs *ControllerServer) ControllerUnpublishVolume(ctx context.Context, req *
 // are supported.
 func (cs *ControllerServer) ValidateVolumeCapabilities(ctx context.Context, req *csi.ValidateVolumeCapabilitiesRequest) (*csi.ValidateVolumeCapabilitiesResponse, error) {
 	if req == nil {
-		return nil, status.Errorf(codes.InvalidArgument, "volume capabilities request is nil")
+		return nil, status.Errorf(codes.InvalidArgument, "ValidateVolumeCapabilities() - request is nil")
 	}
 
-	if req.VolumeId == "" {
-		return nil, status.Error(codes.InvalidArgument, "ValidateVolumeCapabilities() - Volume ID is nil")
+	volumeID := req.GetVolumeId()
+	if volumeID == "" {
+		return nil, status.Error(codes.InvalidArgument, "ValidateVolumeCapabilities() - VolumeId is nil")
 	}
 
 	if req.VolumeCapabilities == nil {
```

Comment:
can you call `req.GetVolumeCapabilities()` function here

Reply:
Legit